lib/BackgroundJobs/SchedulerJob.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+ * Copyright (c) 2021. The Nextcloud Recognize contributors.
+ *
+ * This file is licensed under the Affero General Public License version 3 or later. See the COPYING file.
+ */
+
+namespace OCA\Recognize\BackgroundJobs;
+
+use OCA\Recognize\Service\Logger;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\IJobList;
+use OCP\BackgroundJob\TimedJob;
+use OCP\IDBConnection;
+use Psr\Log\LoggerInterface;
+
+class SchedulerJob extends TimedJob {
+	public const INTERVAL = 30 * 60; // 30 minutes
+	public const ALLOWED_MOUNT_TYPES = [
+		'OC\Files\Mount\LocalHomeMountProvider',
```

Comment:
```suggestion
		'OC\Files\Mount\LocalHomeMountProvider',
		'OC\Files\Mount\ObjectHomeMountProvider',
```

To also cover object storage